### PR TITLE
Fix version regex in monitor script.

### PR DESCRIPTION
**Pull request by ghost**
Recent versions have double-digit last number (e.g. `1.9.14`) and were not being picked up by monitor.bisq.network

---

### Test results:

**Grep version numbers from a test file with this fix:**

`journalctl -u bisq -n 500 | grep -Eo "ersion [0-9]*\.[0-9]*\.[0-9]*" | cut -d " " -f 2 | sort | uniq -c`

```
      1 1.4.0
      1 1.5.0
      2 1.5.2
      2 1.5.5
      2 1.5.7
      2 1.6.0
      2 1.6.3
      2 1.6.5
      2 1.7.2
      2 1.7.4
      2 1.8.0
      2 1.8.1
      2 1.8.3
      2 1.9.0
      2 1.9.10
      2 1.9.11
      2 1.9.13
      2 1.9.3
      2 1.9.5
      2 1.9.6
      2 1.9.7

```

**Grep version numbers from a test file without this fix:**

`journalctl -u bisq -n 500 | grep -Eo "ersion [0-9].[0-9].[0-9]" | cut -d " " -f 2 | sort | uniq -c`

```
      1 1.4.0
      1 1.5.0
      2 1.5.2
      2 1.5.5
      2 1.5.7
      2 1.6.0
      2 1.6.3
      2 1.6.5
      2 1.7.2
      2 1.7.4
      2 1.8.0
      2 1.8.1
      2 1.8.3
      2 1.9.0
      6 1.9.1
      2 1.9.3
      2 1.9.5
      2 1.9.6
      2 1.9.7
```

### CODE REVIEW

File: scripts/journalscraper.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ while true;
 do
 	now=$(date +"%F %T")
 
-	journalctl -u bisq-pricenode --since="$last" --until="$now" | grep -Eo "getAllMarketPrices.*bisq/[0-9].[0-9].[0-9]" | cut -d / -f 2 | sort | uniq -c | while read -r line; do
+	journalctl -u bisq-pricenode --since="$last" --until="$now" | grep -Eo "getAllMarketPrices.*bisq/[0-9]*\.[0-9]*\.[0-9]*" | cut -d / -f 2 | sort | uniq -c | while read -r line; do
```
Comment (Emzy) [inline scripts/journalscraper.sh]: I would do "+" instead of "*". That's a more exact match.
"getAllMarketPrices.*bisq/[0-9]+\.[0-9]+\.[0-9]+"

**Review by Emzy — CHANGES_REQUESTED** [review]

File: scripts/journalscraper.sh (hunk 1 above)
Comment (ghost) [reply scripts/journalscraper.sh]: Change applied, thanks!  Is it ok to match the period explicitly rather than any char?

Comment (Emzy) [reply scripts/journalscraper.sh]: Should be fine. As long as we don't change the versioning schema.

**Review by Emzy — APPROVED** [review]
Tested ACK

**Review by gabernard — APPROVED** [review]
utACK

**Pull request merged**